README.md

```diff
@@ -26,9 +26,9 @@ with an authorization response.
 
 ## Building the SDK ##
 ### Everlaw ###
-In the `servers` repository, the root-level `pom.xml` should have sharefile-api listed as a dependency with the current version. It should also be listed in the `pom.xml` for `web`, without version. On build, it will look for the artifact in your local maven repository, or in the archiva repository on EFS if it doesn't find it locally.
+In the `servers` repository, the root-level `pom.xml` should have sharefile-api listed as a dependency with the current version. It should also be listed in the `pom.xml` for `web`, without version. On build, it will look for the artifact in your local maven repository, or in github packages if it doesn't find it locally.
```

Comment:
Was the previous doc just out of date?

Reply:
The previous readme was way out of date, I don't think the build system has used EFS for this package in years. The sharefile sdk package was uploaded to GH packages in 2021. We might not even be using EFS for anything in the build process at all anymore